src/components/rhelView/rhelView.js

```diff
@@ -35,7 +35,18 @@ class RhelView extends React.Component {
 
     return (
       <PageLayout>
-        <PageHeader>{t('curiosity-view.rhel', helpers.UI_DISPLAY_CONFIG_NAME)}</PageHeader>
+        <PageHeader title={t('curiosity-rhelView.title', helpers.UI_DISPLAY_CONFIG_NAME)}>
+          <p>
+            {`${t('curiosity-rhelView.subtitle')} `}
+            <a
+              href="https://access.redhat.com/documentation/en-us/subscription_central/2020-04/html/getting_started_with_subscription_watch/con-how-does-subscriptionwatch-show-data_assembly-opening-subscriptionwatch-ctxt/"
```

Comment:
I don't see anything wrong with having actual links.  I think our linter and React might need to be adjusted to not require the rel attribute, and we could use a place to store urls or other data that might be reused.  How would Pendo handle it?

Reply:
No, they do not.

As discussed PF4's button component and  minor Googling on why Travis is breaking should be enough of an indicator. 

This is one of those times **where you need to adapt, not the other way around.** PF4

Reply:
Pendo's aspect I'll defer to @rblackbu . We have no plans to store external urls for the appearance of convenience if Pendo is already taking over.

Reply:
"This is one of those times **where you need to adapt, not the other way around.**"

Bold language choice.  (No pun intended.)  Do we have some sort of history that I don't know about?

Obviously I have to comply with whatever you want in your project here.  I was just bringing fresh perspective.

Reply:
heh, indeed there is history. thinking it may be more efficient to setup a weekly get together over the next month or few to go over and answer questions